configure.ac

```diff
@@ -147,7 +147,12 @@ fi
 # HTOP_CHECK_SCRIPT(LIBNAME, FUNCTION, DEFINE, CONFIG_SCRIPT, ELSE_PART)
 m4_define([HTOP_CHECK_SCRIPT],
 [
-   htop_config_script=$([$4] --libs 2> /dev/null)
+   if test ! -z "m4_toupper($HTOP_[$1]_CONFIG_SCRIPT)"; then
```

Comment:
I used
m4_toupper($HTOP_[$1]_CONFIG_SCRIPT)
because I think it is more readable, but I can change to
$HTOP_[]m4_toupper($1)_CONFIG_SCRIPT
if you want